docs: Update website to include SIGHUP, SIGTERM, and SIGINT behaviour (#3140)
hashicorp/boundary: website/content/docs/configuration/controller.mdx

  are anything specified by Go's [ParseDuration()](https://golang.org/pkg/time/#ParseDuration) method. Only
  used when an `ops` listener is set and the Controller is present. Default is 0 seconds.

## Signals

> On `SIGHUP`, a controller will reload its configuration file to pick up any updates to its `database url`. All other updated values will be ignored.

Reviewer: ```suggestion
The `SIGHUP` signal causes a controller to reload its configuration file to pick up any updates to the `database url` value. Any other updated values are ignored.
```

> `SIGTERM` or `SIGINT` will initiate graceful shutdown on a controller. Controller graceful shutdown
> will close listeners and servers before shutting down.

Reviewer: ```suggestion
The `SIGTERM` and `SIGINT` signals initiate a graceful shutdown on a controller. A graceful shutdown closes listeners and servers before shutting down the controller.
```
